app/models/reliefModels.scala

```diff
@@ -141,7 +141,7 @@ object ReliefsTaxAvoidance {
         (JsPath \ "taxAvoidance").write[TaxAvoidance] and
         (JsPath \ "periodStartDate").write[LocalDate] and
         (JsPath \ "periodEndDate").write[LocalDate] and
-        (JsPath \ "timeStamp").write[DateTime](MongoJodaFormats.Implicits.jotDateTimeFormat)
+        (JsPath \ "timeStamp").write[DateTime](MongoDateTimeFormats.Implicits.mdDateTimeFormat)
```

Comment:
would use MongoDateTimeFormats.tolerantDateTimeFormat as implicits is not relevant here